src/lib.rs

```diff
@@ -44,6 +46,8 @@ mod definitions;
 mod params;
 mod table;
 
+#[cfg(feature = "ansi")]
```

Comment:
The ansi should enable alloc.

Reply:
https://github.com/alacritty/vte/pull/85/commits/9f3c6850adc997df324fe79b76ac624e6c6b6e29